### PR TITLE
Multi target and friends

### PR DESCRIPTION
**Pull request by fmherschel**
Next week stone

### CODE REVIEW

**Comment by fmherschel** [comment]
Most changes are related to the man-pages

File: SAPHana/man/SAPHanaSR-ScaleOut.7 [hunk 1]
```diff
@@ -253,6 +252,9 @@ that one does not interfere with the Linux cluster. All three clusters
 11. The RAs SAPHanaController and SAPHanaToplogy need to be installed on all
 cluster nodes, even the majority maker.
 .PP
+11. The Linux cluster needs to be up an running to allow HA/DR provider events
```
Comment (angelabriel) [inline SAPHana/man/SAPHanaSR-ScaleOut.7]:
```suggestion
11. The Linux cluster needs to be up and running to allow HA/DR provider events
```

File: SAPHana/man/SAPHanaSR.py.7 [hunk 2]
```diff
@@ -155,6 +153,10 @@ in memory and on disk (in persistence).
 .PP
 4. If the hook provider should be pre-compiled, the particular Python version that comes with SAP HANA has to be used.
 .PP
+5. The Linux cluster needs to be up an running to allow HA/DR provider events
```
Comment (angelabriel) [inline SAPHana/man/SAPHanaSR.py.7]:
```suggestion
5. The Linux cluster needs to be up and running to allow HA/DR provider events
```

File: SAPHana/man/SAPHanaSR_maintenance_examples.7 [hunk 3]
```diff
@@ -215,6 +373,10 @@ left-over migration constraints, and resource failures as well as the HANA
 landscape status, and the HANA SR status.
 .PP
 \fB*\fR Maintenance attributes for cluster, nodes and resources must not be mixed.
+.PP
+\fB*\fR The Linux cluster needs to be up an running to allow HA/DR provider events being written into CIB attributes.
```
Comment (angelabriel) [inline SAPHana/man/SAPHanaSR_maintenance_examples.7]:
```suggestion
\fB*\fR The Linux cluster needs to be up and running to allow HA/DR provider events being written into CIB attributes. 
```

File: SAPHana/man/SAPHanaSrMultiTarget.py.7 [hunk 4]
```diff
@@ -202,6 +232,8 @@ in memory and on disk (in persistence).
 .PP
 4. If the hook provider should be pre-compiled, the particular Python version that comes with SAP HANA has to be used.
 .PP
+5. The Linux cluster needs to be up an running to allow HA/DR provider events being written into CIB attributes. The current HANA SR status might differ from CIB srHook attribute after cluster maintenance.
```
Comment (angelabriel) [inline SAPHana/man/SAPHanaSrMultiTarget.py.7]:
```suggestion
5. The Linux cluster needs to be up and running to allow HA/DR provider events being written into CIB attributes. The current HANA SR status might differ from CIB srHook attribute after cluster maintenance.
```

**Review by angelabriel — COMMENTED** [review]
I have found some small typos in the man pages. Can we change that before I merge the changes and build the package?
Thanks

**Comment by fmherschel** [comment]
Hi Angela I used the button to submit your proposed changes. However once I run "gut pull" I do not get them back into my repo ?!?

**Comment by angelabriel** [comment]
try it again, as I just merged the changes.

**Pull request merged**